src/dom-parts/directives.js

```diff
@@ -102,10 +104,37 @@ export class SpreadAttributesDirective extends Directive {
     }
 }
 
+/**
+ * Renders an attribute as key="value" if a condition is truthy
+ */
+export class OptionalAttributeDirective extends Directive {
```

Comment:
@quarkus can we also add the stringify method (plus tests) to be able to use this feature in SSR mode?